### PR TITLE
Update documentation pages on the website 

### PR DESCRIPTION
**Pull request by OndroMih**
Depends on https://github.com/eclipse-ee4j/glassfish/pull/24237

- use a newer Antrun plugin (works with target element)
- generate docs for GF 6 and 7, 7 is the latest

WARNING: Build fails until org.glassfish.docs:distribution version 6.2.5 and 7.0.0 are deployed to Maven repo

### CODE REVIEW

**Review by dmatej — APPROVED** [review]

**Comment by dmatej** [comment]
https://ondromih.github.io/glassfish/docs/#current 
The move from development version to current version (perhaps better mark them by numbers than by dev/current/5.1), that will be in this PR which follows the https://github.com/eclipse-ee4j/glassfish/pull/24237 , am I right?

**Comment by OndroMih** [comment]
> https://ondromih.github.io/glassfish/docs/#current The move from development version to current version (perhaps better mark them by numbers than by dev/current/5.1), that will be in this PR which follows the #24237 , am I right?

The PR #24237 can be merged first, it only updated the News and Download pages, doesn't break the build. The preview at https://ondromih.github.io/glassfish/ includes only that PR, not this PR.

This PR will fix the https://ondromih.github.io/glassfish/docs/#current, which will show the GF 7 docs. I'll generate a new preview of that once #24237 is merged. I can't generate 2 previews at the same time, I would have to create to forks of the GF repo for that.

**Comment by dmatej** [comment]
Why draft again?

**Comment by OndroMih** [comment]
> Why draft again?

Not anymore :) I moved it back to draft before because I found out that the artifacts aren't still present in the Maven repo. As of now, the artifacts are already there and this PR can be merged after approval.

**Pull request merged**
